### DISCUSSION TITLE
03/12/2023 it works

### DISCUSSION THREAD

**Discussion by coronacharly** [Daksh777/SpotifyNoPremium#115] [Show and tell]
hello

just to confirm that as of the current date 12/13/2023 it works without problems. If you have any failure, just follow the instructions and if you still have a failure, then you are doing something wrong.

Thank you @Daksh777 for contributing this feature for Spotify

For those who want to know if it can be extended to a cell phone, no it cannot but there are other tools with which a similar function can be enabled, it exists for Android and iOS.
If you need help on the subject you can contact me.
I would like to clarify that I am offering my help for free and in good faith.

**Comment by MirdjusCZ** [comment]
I am not very into these thinks, but if I install the .bat file and I run it, it starts to write something in the command line and then what? It writes press any key to continue, but thats all. What I have to do to actually install it? I thought it will install the spotify but it didnt.
Thanks for response.

**Comment by coronacharly** [comment]
The Spotify app should not be downloaded from the Microsoft store. I have a zip with the Spotify app, the two .bat files you will need to install and update on terabox or Mega. If you are interested, I will gladly share the link with you.


El 14 dic 2023, a la(s) 2:06 p.m., MirdjusCZ ***@***.***> escribió:

﻿

I am not very into these thinks, but if I install the .bat file and I run it, it starts to write something in the command line and then what? It writes press any key to continue, but thats all. What I have to do to actually install it? I thought it will install the spotify but it didnt.
Thanks for response.

**Comment by MirdjusCZ** [comment]
I dont have any idea, what I have to do now, so if you can teach me, it would be really nice.

**Comment by coronacharly** [comment]
here is a zip with the instructions and the necessary files. If you have any difficulties, don't hesitate to let me know.
https://teraboxapp.com/s/1qBFkg_gLF9L_QOuv7-4cgg
________________________________
De: MirdjusCZ ***@***.***>
Enviado: viernes, 15 de diciembre de 2023 07:28 a. m.
Para: Daksh777/SpotifyNoPremium ***@***.***>
Cc: coronacharly ***@***.***>; Author ***@***.***>
Asunto: Re: [Daksh777/SpotifyNoPremium] 03/12/2023 it works (Discussion #115)


I dont have any idea, what I have to do now, so if you can teach me, it would be really nice.

**Comment by MirdjusCZ** [comment]
bro, it has some virus...

**Reply by coronacharly** [reply]
Not any virus. You can verify the files on this very useful page: https://www.virustotal.com/gui/home/upload here you can verify files or pages directly.
Or you can also create a virtual machine to check the files without them having access to your main PC.
There are several ways to check it but I really have no bad intentions.
